db/deploy/removed-hotel.sql

```diff
@@ -0,0 +1,252 @@
+-- Deploy rams-core:removed-hotel to pg
+
+BEGIN;
+
+CREATE SCHEMA sqitch;
```

Comment:
Yeah, looks like apgdiff exploded a little. Please go ahead and remove all the extra cruft like the comments and the create schema command.